extensions/default/src/getCustomizationModule.tsx

```diff
@@ -312,7 +312,7 @@ export default function getCustomizationModule({ servicesManager, extensionManag
             },
             {
               id: 'showInOtherMonitor',
-              label: 'Launch On Other Monitor',
+              label: 'Launch On Second Monitor',
```

Comment:
It is actually supposed to be other monitor since there might be three monitors, and it might be executed on the second monitor already, causing it to launch on the first monitor.